conf/machine/include/imx-base.inc

```diff
@@ -165,6 +165,10 @@ MACHINEOVERRIDES_EXTENDER:mx8dxl:use-nxp-bsp = "imx-generic-bsp:imx-nxp-bsp:mx8-
 ### Mainline BSP specific overrides
 #######
 
+MACHINEOVERRIDES_EXTENDER:mxs:use-mainline-bsp   = "imx-generic-bsp:imx-mainline-bsp:mxs-generic-bsp:mxs-mainline-bsp"
+
```

Comment:
The `mxs` can be dropped. It is equivalent to `mx5` or `mx6` which intends to group a set of machines. In this case, it'd group `mx23` and `mx28`.